scripts/docker-eth

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+mkdir -p ~/.ethereum ~/.web3
+
+# check if sudo is necessary
+if ! id -nG $(whoami)|grep -qw "docker"; then SUDO='sudo'; else SUDO=''; fi
+
+$SUDO docker run --rm -it \
+           --net=host \
```

Comment:
By the way: the need for `--net=host` makes me unhappy.

It results in publishing the jsonrpc interface on `0.0.0.0:8545`, which is pretty insecure! It is necessary though in order to make the UPnP punched nat traversal work:

Without `--net=host`:
```
  ℹ  10:19:07.397|p2p  UPnP device not found.
```

There is an issue that seems to address this issue: (https://github.com/docker/docker/issues/3043 and in particular https://github.com/docker/docker/issues/3043#issuecomment-72387449), but I still could not get it to work.

If UPnP nat traversal is considered less important than the wide open rpc interface, then `--net=host` should be replaced by:
```
    -p 127.0.0.1:8545:8545 \                                                    
    -p 0.0.0.0:30303:30303 \   
```

@chriseth let me know, if you want this addressed!